<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>543</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_3" rowstretch="0,0,1,0">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_Username"/>
      </item>
      <item row="1" column="3">
       <widget class="QLineEdit" name="lineEdit_Password">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Server</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLineEdit" name="lineEdit_Port"/>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_Server"/>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Password</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Port</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Username</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>FTP</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="1" column="0">
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Local Path</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLineEdit" name="lineEdit_LocalPath"/>
           </item>
           <item row="4" column="0" colspan="2">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>ex) /home/home.exe</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>FTP Path</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QToolButton" name="toolButton_LocalPath">
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLineEdit" name="lineEdit_FTPPath"/>
           </item>
           <item row="5" column="0" colspan="2">
            <layout class="QGridLayout" name="gridLayout_8">
             <item row="0" column="3">
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="0" column="1">
              <widget class="QPushButton" name="pushButton_Download">
               <property name="text">
                <string>Download</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="0" column="2">
              <widget class="QPushButton" name="pushButton_Upload">
               <property name="text">
                <string>Upload</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="label_Byte">
        <property name="text">
         <string>0 / 0</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QProgressBar" name="progressBar">
        <property name="value">
         <number>24</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QTextEdit" name="textEdit_Console"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>543</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lineEdit_Server</tabstop>
  <tabstop>lineEdit_Port</tabstop>
  <tabstop>lineEdit_Username</tabstop>
  <tabstop>lineEdit_Password</tabstop>
  <tabstop>lineEdit_LocalPath</tabstop>
  <tabstop>toolButton_LocalPath</tabstop>
  <tabstop>lineEdit_FTPPath</tabstop>
  <tabstop>pushButton_Download</tabstop>
  <tabstop>pushButton_Upload</tabstop>
  <tabstop>textEdit_Console</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
